.github/workflows/compat-check-latest-latest.yml

```diff
@@ -0,0 +1,20 @@
+name: Compat Check w/PL - latest/latest
+
+on:
+  schedule:
+    - cron: "0 3 * * 1-5"  # Run daily at 3am Mon-Fri
```

Comment:
Any reason 3am is used? Or is it arbitrary?

Reply:
It's mostly arbitrary, but plugin-test-matrix workflows are triggered at midnight, so I'm thinking we should let them some time to finish. At 3 o'clock Actions should be least busy too I reckon.